osquery/config/parsers/decorators.cpp

```diff
@@ -237,7 +238,6 @@ void runDecorators(DecorationPoint point,
 
   // Abstract the use of the decorator parser API.
   auto dp = std::dynamic_pointer_cast<DecoratorsConfigParserPlugin>(parser);
-  WriteLock lock(DecoratorsConfigParserPlugin::kDecorationsMutex);
```

Comment:
This will not work since `db->load_` and others are now unprotected. Perhaps two mutexes are needed, 1 to protect the config/schedule, and another to protect the decorator content.